test/jdk/javax/swing/JFileChooser/JFileChooserSetLocationTest.java

```diff
@@ -209,7 +210,11 @@ private static void hitKeys(int... keys) {
     }
 
     public static void createUI() {
-        frame = new JFrame();
+        Dimension screenSize = Toolkit.getDefaultToolkit().getScreenSize();
+
+        int width = (int) screenSize.getWidth() / 2;
+        int height = (int) screenSize.getHeight() / 2;
```

Comment:
This is not the width and height, this is the kind of point(x,y) where the frame will be located.

Reply:
Yeah, I got it now. Updated.